backend/pkg/server/server.go

```diff
@@ -36,10 +36,14 @@ func New(
 	restaurantHandler := restaurantHandler{restaurantService:restaurantService}
 
 	r.Route("/api", func(r chi.Router) {
-		r.Route("/restaurant", func(r chi.Router) {
+		r.Route("/exists", func(r chi.Router) {
 			r.Get("/{restaurantID}", restaurantHandler.exists)
 		})
 
+		r.Route("/restaurant", func(r chi.Router) {
```

Comment:
jak masz już otwarty route, to można do niego dopisać wiele endpointów, tutaj raczej powinno być 
```
r.Route("/restaurants", func(r chi.Router) {
    r.Get("/", restaurantHandler.listRestaurants)
    r.Get("/{restaurantID}", restaurantHandler.restaurantMenu)
}
```

Reply:
chociaż i tak nie do końca, raczej kolejny route z argumentem `{restaurandID}`, i tam udostępnione zasoby przypisane do danej restauracji, m.in. właśnie menu restauracji
ale na razie może tak być